app/api/logging/log_formatters.py

```diff
@@ -29,7 +29,7 @@ def format(self, record: logging.LogRecord) -> str:
         super(JsonFormatter, self).format(record)
 
         output = {
-            key: value
+            key: str(value)
```

Comment:
Removing the masking in a prior PR makes the UUID conversion fail, not sure how I missed this when testing removing that (probably something with the container kept it working?)